distributed/comm/tcp.py

```diff
@@ -145,6 +145,8 @@ class TCP(Comm):
     An established communication based on an underlying Tornado IOStream.
     """
 
+    max_shard_size = dask.utils.parse_bytes(dask.config.get("distributed.comm.shard"))
```

Comment:
This seems to be [`64 MiB`]( https://github.com/dask/distributed/blob/83e209b933430de63539a939a6499e79370d7b91/distributed/distributed.yaml#L172 ). Do we want to limit it to that? Would it be worthwhile to have a different value for TCP specifically?

Reply:
This PR doesn't change the behavior for TCP (this value is what is used by default for everything in to_frames).  All it does is make sure that TLS has a possibly different value that is capped by the max int size, which I think was the intent of the other PR.

Reply:
Got it. So this was 64 MiB before?

Reply:
Yup.